tests/enforcement/test_quality_ratchet.py

```
@@ -0,0 +1,321 @@
+
+try:
+    spec = importlib.util.spec_from_file_location("quality_ratchet", script_path)
+    quality_ratchet = importlib.util.module_from_spec(spec)
+    spec.loader.exec_module(quality_ratchet)
+
+    load_history = quality_ratchet.load_history
+    save_history = quality_ratchet.save_history
+    detect_degradation = quality_ratchet.detect_degradation
+    calculate_moving_average = quality_ratchet.calculate_moving_average
+    find_peak_quality = quality_ratchet.find_peak_quality
+except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)